### PR TITLE
test: add test to `feat/dynamic-party-card-price`

### PR DESCRIPTION
**Pull request by 0xble**

### CODE REVIEW

File: test/authorities/BondingCurveAuthority.t.sol [hunk 1]
```diff
@@ -1,60 +1,200 @@
...
+import { Party } from "../../contracts/party/Party.sol";
+import { PartyFactory } from "../../contracts/party/PartyFactory.sol";
 import { BondingCurveAuthority } from "../../contracts/authorities/BondingCurveAuthority.sol";
-import { ArbitraryCallsProposal } from "../../contracts/proposals/ArbitraryCallsProposal.sol";
-
-contract BondingCurveAuthorityTest is SetupPartyHelper {
-    constructor() SetupPartyHelper(false) {}
+import "../TestUtils.sol";
 
+contract BondingCurveAuthorityTest is TestUtils {
```
Comment (arr00) [inline test/authorities/BondingCurveAuthority.t.sol:10]: Why would we not use `SetupPartyHelper`?

Comment (0xble) [reply test/authorities/BondingCurveAuthority.t.sol:10]: The Party is being deployed through `BondingCurveAuthority.createParty()`. Guess we could override `setUp()` though

Comment (arr00) [reply test/authorities/BondingCurveAuthority.t.sol:10]: It still exposes all the other contracts needed so I think it worth it.

**Review by arr00 — APPROVED** [review]
Looks great

**Pull request merged**